<launch>

  <arg name="use_sim_time" default="true" />
  <arg name="variant" default="" />

    <!-- set use_sim_time flag -->
  <group if="$(arg use_sim_time)">
    <param name="/use_sim_time" value="true" />
  </group>

  <node name="target_mover" pkg="task_02_gazebo_resources" type="target_mover" respawn="false" output="screen" >

    <rosparam file="$(find task_02_gazebo_resources)/config/target_mover_$(arg variant).yaml" />

    <!-- subscribers -->
    <remap from="~diagnostics_in" to="/task_02_wrapper/diagnostics" />

  </node>

</launch>
